docroot/profiles/custom/cgov_site/modules/custom/cgov_yaml_content/README.md

```diff
@@ -7,16 +7,13 @@ This tool is especially helpful for expediting demo preparations, test environme
 
 ## How do you use me?
 
-First create XXX.content.yml files in the cgov_yaml_content/content directory. Then run 'drush ycim cgov_yaml_content' from the command line after installing Drupal (inside the Docker container).
+First create XXXXX.content.yml files in the cgov_yaml_content/content directory. Then run 'drush ycim cgov_yaml_content' from the command line after installing Drupal (inside the Docker container).
 
 All of the files in the content directory are sorted alphabetically before loading. So if you have dependencies across files, you need to make sure the files are named such that the order is maintained. As such we are following the Basic Programming style of line numbering. (Numbers start the filenames and you leave enough space between the next group to fit new items in)
```

Comment:
What about the note when adding a new file to pick a number halfway between 2 other files.